README.md

```diff
@@ -1,2 +1,299 @@
+
+Example for MySQL:
+```yaml
+wso2:
+    config:
+        userstore:
+            file:
+                enabled: false
+            rdbms:
+                url: "jdbc:mysql://{hostname}:{port}/userdb"
+                username: "<username>"
+                password: "<password>"
+                jdbc:
+                driver: "com.mysql.jdbc.Driver"
+                poolParameters:
+                    maxActive: 50
+                    maxWait: 60000
+```
+> **Note**: When using RDBMS, the JDBC driver must be added to the `<SERVER_HOME>/lib` folder. To achieve this, you need to build a custom server image. Follow the steps below:
+ 1. Create the Dockerfile 
+    - Base Images:
+        * MI: `wso2/wso2mi:4.3.0`
+        * ICP: `wso2/wso2-integration-control-plane:1.0.0`
+    
+    Example for MySQL:
+
+    ```yaml
+        FROM <base-image>
+        USER root
+        RUN apt-get update && \
+            apt-get install -y wget && \
+            apt-get clean && \
+            rm -rf /var/lib/apt/lists/*
+        ENV WSO2_SERVER_HOME=/home/wso2carbon/wso2-integration-control-plane-1.0.0
+        RUN mkdir -p ${WSO2_SERVER_HOME}/lib
+        ARG JDBC_DRIVER_URL=https://dev.mysql.com/get/Downloads/Connector-J/mysql-connector-j-8.0.33.tar.gz
+        RUN wget -O /tmp/mysql-connector.tar.gz "${JDBC_DRIVER_URL}" && \
+            tar -xzf /tmp/mysql-connector.tar.gz -C /tmp && \
+            find /tmp -name "mysql-connector*.jar" -exec cp {} ${WSO2_SERVER_HOME}/lib/ \; && \
+            rm -rf /tmp/mysql-connector.tar.gz
+        RUN chown -R wso2carbon:wso2 ${WSO2_SERVER_HOME}/lib && \
+            chmod -R 755 ${WSO2_SERVER_HOME}/lib
+        USER wso2carbon
+    ```
+ 2. Build the image:
+    ```yaml
+    docker build -t customized-wso2-img:1.0.0 .
+    ```    
+ 3. Update the following configurations in `values_local.yaml`:
+    ```yaml
+    containerRegistry: ""
+    wso2:
+        deployment:
+            image:
+                repository: "customized-wso2-img"
+                tag: "1.0.0"
+                pullPolicy: IfNotPresent
+    ```   
+
+> **Note**: For integration development, if you are using the [MI VSCode extension](https://marketplace.visualstudio.com/items/?itemName=WSO2.micro-integrator), you can add JDBC drivers by placing the JAR file in the `<PROJECT_DIR>/deployment/libs` folder and then clicking "Create Docker Image" under Deployment Options. This will build the Docker image and push it to the local registry. 
+
+<div style="display: flex; justify-content: space-around; align-items: center;">
+    <div style="display: flex; justify-content: space-around; align-items: center;">
+        <figure style="width: 40%; height: 500px; text-align: center;">
+            <img src="resources/project-structure.png" alt="Project Structure" />
```

Comment:
Have we missed uploading these images?

Reply:
uploaded by https://github.com/wso2/helm-mi/pull/18/commits/81752588a0b25ac6187cb8296525d2bd4bc2464a